src/vconsole/vconsole-setup.c

```diff
@@ -190,7 +193,7 @@ static int font_load_and_wait(const char *vc, const char *font, const char *map,
  * font. It also allows to restart systemd-vconsole-setup.service,
  * to apply a new font to all VTs.
  */
-static void font_copy_to_all_vcs(int fd) {
+static void font_copy_to_all_vcs(int fd, int vc_max) {
```

Comment:
hmm, given that this parameter cannot be realistically negative, this should better be an unsigned...